src/flashsystem-dashboard.tsx

```diff
@@ -85,20 +87,39 @@ const FlashsystemDashboard: React.FC<ODFDashboardProps> = (props) => {
   );
 };
 
-export const FlashsystemDashboardPage: React.FC<any> = (props) => {
-  const { t } = useTranslation("plugin__ibm-storage-odf-plugin");
-  const allPages = [
-    {
-      href: "",
-      name: t("Overview"),
-      component: FlashsystemDashboard,
-    },
-  ];
-  return (
-    <>
-      <HorizontalNav pages={allPages} {...props} />
-    </>
-  );
+export const FlashsystemDashboardPage: React.FC<FlashsystemDashboardPageProps> =
```

Comment:
FlashsystemDashboardPage is not used any more. I think we can add page heading in FlashsystemDashboard